app/utils/steam/steamcmd/wrapper.py

```diff
@@ -262,56 +323,49 @@ def setup_steamcmd(
             )
             runner.message(f"Symlink source : {symlink_source_path}")
             runner.message(f"Symlink destination: {symlink_destination_path}")
-            if os.path.exists(symlink_destination_path):
+            if self.is_junction_or_link(symlink_destination_path): # Symlink/junction exists
                 runner.message(
                     f"Symlink destination already exists! Please remove existing destination:\n\n{symlink_destination_path}\n"
                 )
-            else:
+                answer = show_dialogue_conditional(
+                "Re-create Symlink?",
+                "An existing symlink already exists."
+                " Would you like to delete and re-create the symlink?",
+                f"Existing symlink: {symlink_destination_path}",
+                "The symlink makes SteamCMD download mods to the local mods folder"
+                + " and is required for SteamCMD mod downloads to work correctly."
```

Comment:
Maybe move the explanation of what the symlink/junction is for into main message, and the existing symlink path into details.


```suggestion
                "The symlink makes SteamCMD download mods to the local mods folder"
                + " and is required for SteamCMD mod downloads to work correctly.",
                f"Existing symlink: {symlink_destination_path}" ...
```